src/canvas.ts

```diff
@@ -184,15 +186,16 @@ export class CanvasAssignmentWrapper {
         );
     }
     /** Returns short description of a canvas assignment with the specified format
-     *  @returns: String of pattern COURSE NAME CODE: Assignment Name.
+     *  @returns: String of pattern [COURSE NAME CODE]: Assignment Name.
      * */
     get title(): string {
         let desc = "";
         //Lets start by getting the course name and a colon
-        if (this.canvas_general_infos.context_name){
+        if (this.canvas_course_name){
+            desc += "" + this.canvas_course_name.toLowerCase().trim()
```

Comment:
What are the empty quotes for? Is `this.canvas_course_name.toLowerCase().trim()` not a string? If it's not then `String()` is probably a better way to do this. If it is a string (which is looks like should be), then I don't know what those quotes are for. Looks like this is done a few other places too.